pkg/util/clusterlease.go

```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+	"context"
+
+	coordinationv1 "k8s.io/api/coordination/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1"
```

Comment:
Please give this import an import alias `clusterv1alpha1`.

Reply:
Sure.